src/lib/rnp.cpp

```diff
@@ -4049,18 +4049,15 @@ rnp_key_get_revocation(rnp_ffi_t        ffi,
 
 rnp_result_t
 rnp_key_export_revocation(rnp_key_handle_t key,
-                          rnp_output_t     output,
+                          rnp_output_t     external_output,
```

Comment:
Why do we need parameter name change here?

Reply:
To disambiguate with the other "output" entity which was either the original, or the armoring one.
I reverted this change while addressing your other comment.